exercises/creating_python_actions/exercise.js

```diff
@@ -6,7 +6,7 @@ exercise.requireSubmission = false
 exercise.addVerifyProcessor(function (cb) {
   shell.exec('wsk action get hello-world-python', {silent: true}, (code, stdout, stderr) => {
     const exists = (code === 0)
-    const uses_python = (stdout.match('"kind": "python"'))
+    const uses_python = (stdout.match(/\"kind\": \"python:?[\d.]+?"/))
```

Comment:
The regular expression should also cover python without version, i.e.`"python"`, which is a candidate value of `"kind"`. The updated regular expression cannot cover it. I will suggest to use `stdout.match(/\"kind\": \"python:?[\d.]?"/)` instead.